src/delfic3/migrations/0001_initial.py

```diff
@@ -1,4 +1,4 @@
-# Generated by Django 3.2.8 on 2021-12-12 19:55
```

Comment:
I would expect this to generate a new migration on top of the initial, did you remove them locally before or does django work in this way?

Reply:
Ah, I see there is no content change in this file so that's all good ;). I'm still curious why it regenerated this file

Reply:
I did run into some issues and then I removed the migration files and recreated them :). Django doesn't modify existing migration files